### DISCUSSION TITLE
How to install on Mac with Apple Silicon (M1)?

### DISCUSSION THREAD

**Discussion by mekeor** [audacity/audacity#3126] [General]
How to install Audacity on a Mac computer with an Apple Silicon (M1) processor? Apparently, it works with Rosetta.

1. Should the Intel-binary be used? (E.g. https://github.com/audacity/audacity/releases/download/Audacity-3.1.3/audacity-macos-3.1.3-Intel.dmg)
2. Will the user be asked if they want to install Rosetta if it's not installed yet?
3. Will Rosetta be executed automatically when Audacity is launched?
4. I.e. is the installation flow just he same as on Intel-Macs?

**Comment by LWinterberg** [comment]
For 3.2 and onwards, there will be native macOS arm64 versions as well as universal binaries. You can download the current alpha on https://audacityteam.org/nightly

**Reply by shalte1981** [reply]
I wish I had seen the nightly binaries first, but I was able to compile Audacity 3.2.0-alpha-20220623 from source for Apple Silicon and it seems to work well 😊

**Reply by danorton** [reply]
What about the FFmpeg library?

**Reply by crsib** [reply]
It's available from Brew

**Reply by danorton** [reply]
Turns out that the ffmpeg utility by itself more easily did what I wanted: convert stereo to mono. Thanks!

**Comment by thynus** [comment]
A little confused; but it seems to work fine.  

Downloaded Audacity 3.2.0 x86_64 and installed on an Intel MacBook Pro running macOS 11.7.  Installed FFmpeg from the Audacity>Preferences>Libraries>Download link.  FFmpeg was detected.

Repeated above on an m1 Apple Silicon mac mini.  FFmpeg was detected.

Deleted the x86_64 application on the m1.  

Downloaded and installed the ARM64 application from audacityteam.org.  It is listed as an Apple binary in Activity Monitor.  I was about to reinstall the FFmpeg libraries but they were already recognized from the previous x86 installation.  And things work fine?  The magic of Rosetta at work?

**Reply by crsib** [reply]
The binary from the downloads section is "universal", that's it - it contains both Intel and ARM code inside. We will be distributing Audacity in the very same way after dealing with some weird bug with localization
